metadata-ingestion/setup.py

```diff
@@ -465,7 +465,7 @@
     "lookml": looker_common,
     "metabase": {"requests"} | sqlglot_lib,
     "mlflow": {
-        "mlflow-skinny>=2.3.0",
+        "mlflow-skinny>=2.3.0,<2.21.0",
```

Comment:
the comments from the PR description should be inlined as comments here

it should also be clear when we can remove this upper bound

Reply:
good point, done